README.md

```diff
@@ -68,8 +68,23 @@ Background information on CB8 may be found in a number of publications, includin
 
 ### Physical properties
 Due to experimental issues, the upcoming SAMPL6 physical property challenge is being split into two phases.
-The first phase, for SAMPL6, will focus on pKa prediction for the upcoming workshop.
-Then the pKa data for this challenge will be provided to participants and used as part of a second challenge centering on predicting physicochemical properties for 25-50 fragment- and drug-like small molecules that small molecule protein kinase inhibitors (or fragments thereof) -- specifically, distribution coefficients and potentially also solubilities.
+The first phase, for SAMPL6, will focus on pKa prediction for the upcoming workshop. Then the pKa data for this challenge will be provided to participants and used as part of a second challenge centering on predicting distribution coefficients.
+
+#### pKa prediction
+This challenge consists of predicting microscopic and macroscopic pKas of 24 small organic molecules. These fragment-like small molecules are selected for their similarity to kinase inhibitors and for experimental tractability. Our aim is to evaluate how well current pKa prediction methods perform with drug fragment-like molecules through blind predictions. This is the first time a pKa prediction challenge is being conducting as a part of SAMPL.
+
+Three formats of pKa prediction results will be evaluated:
+1. microscopic pKa values and related microstates
+2. microstate populations as a function of pH
+3. macroscopic pKa values
+
+For detailed instructions for pKa challenge: [pKa_challenge_instructions.md](pKa_challenge_instructions.md)
+
+Challenge start date: Oct 24, 2017   
+Challenge submission due: Jan 10, 2018  
+
+#### logD prediction
```

Comment:
Do we want to update the timeline for the logD challenge to say that it will occur in early or mid 2018?

Reply:
I think it would be good to specify that it is happening later at least incase new people to the repository are confused about why there are deadlines for pKa and not logD

Reply:
I added a sentence indicating logD challenge will take place in early 2018.